app/views/automated_tests/_boot.js.erb

```diff
@@ -1,8 +1,36 @@
 <script>
-  jQuery(document).ready(function() {
-    <% if @assignment %>
-      toggle_automated_tests(<%= @assignment.enable_test %>);
-      toggle_tests_tokens(<%= @assignment.unlimited_tokens %>)
-    <% end %>
-  });
+
+  function check_start_of_availability_date(new_start_date) {
+    if (moment(new_start_date, '<%= t('date.formats.js_old_date') %>').isBefore(
+    moment())) {
+      alert('<%= t(:past_start_date_edit_warning) %>');
+    }
+    }
+
+  jQuery(window).load(function() {
```

Comment:
I was only able to get it to work with .load function. I could not get timepicker on Test Framework to work with .ready function.